crl/updater/updater.go

```diff
@@ -279,6 +287,30 @@ func (cu *crlUpdater) tickIssuer(ctx context.Context, atTime time.Time, issuerNa
 	return nil
 }
 
+// tickShardWithRetry calls tickShard repeatedly (with exponential backoff
+// between attempts) until it succeeds or the max number of attempts is reached.
+func (cu *crlUpdater) tickShardWithRetry(ctx context.Context, atTime time.Time, issuerNameID issuance.IssuerNameID, shardIdx int, chunks []chunk) error {
+	crlID := crl.Id(issuerNameID, crl.Number(atTime), shardIdx)
+
+	var err error
+	for i := 0; i < cu.maxAttempts; i++ {
+		// core.RetryBackoff always returns 0 when its first argument is zero.
+		sleepTime := core.RetryBackoff(i, time.Second, time.Minute, 2)
+		if i != 0 {
+			cu.log.Errf(
+				"Generating CRL failed, will retry in %vs: id=[%s] err=[%s]",
```

Comment:
That's because those are returning errors with fmt.Errorf, which respects the %w error-wrapping format operator. This is just logging a string, so it doesn't support %w, so we have to use %s instead.